fastn-core/src/auth/email_password.rs

```diff
@@ -164,3 +202,24 @@ pub(crate) async fn login(
     // https://stackoverflow.com/a/39739894
     fastn_core::auth::set_session_cookie_and_end_response(req, session_id, next).await
 }
+
+fn confirmation_mail_body(link: String) -> String {
+    format!("Use this link to verify your email: {link}")
+}
```

Comment:
Store this in an external file so end user can modify it. Do a fallback in case the file does not exit.

Reply:
This will be done in another PR :+1: